lib/publiccloud/instance.pm

```diff
@@ -34,7 +34,7 @@ has image_id => undef;    # image from where the VM is booted
 has type => undef;
 has region => undef;    # provider region, filled by provider::terraform_apply
 has provider => undef, weak => 1;    # back reference to the provider
-has ssh_opts => '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o LogLevel=ERROR';
```

Comment:
`'-o UserKnownHostsFile=/dev/null'` probably should stay here

Reply:
ee, we should keep checking it. Read the commit message:
> Get rid of -o UserKnownHostsFile=/dev/null and -o StrictHostKeyChecking=no as those might hide real issues and when we expect host key change we use ssh-keyscan to fetch it.